packages/vitest/src/runtime/runners/test.ts

```diff
@@ -36,6 +36,16 @@ export class VitestTestRunner implements VitestRunner {
   onAfterRunSuite(suite: Suite) {
     if (this.config.logHeapUsage && typeof process !== 'undefined')
       suite.result!.heap = process.memoryUsage().heapUsed
+
+    getTasks(suite).forEach((suiteTask) => {
+      if (suiteTask.mode === 'skip') {
+        getTasks(suiteTask).forEach((task) => {
```

Comment:
I don't think there are any tasks here when suite is skipped. We skip collection phase altogether

Reply:
Weird, I was logging the tasks and they were there. The warning is also not happening anymore as reported in the issue.

Reply:
Yeah, the tasks are there but they have `mode: 'skip'`

Reply:
You can see here that we don't collect tests if suite was marked as skipped, so I'm not sure how you tested:

https://github.com/vitest-dev/vitest/blob/00c432fad6c40b4ff21cc45b18e64f29816062db/packages/runner/src/run.ts#L284